cjwstate/modules/types.py

```diff
@@ -13,6 +13,14 @@
 from cjwstate.modules.param_dtype import ParamDType
 from .module_loader import validate_module_spec
 from .param_spec import ParamSpec
+from cjworkbench.i18n import supported_locales
+from babel.messages.pofile import read_po, PoFileError
```

Comment:
Do not import Babel. Medium-term, `renderer` and `fetcher`, which use `cjwstate`, may be distributed without Babel.